devicetypes/smartthings/zigbee-dual-switch.src/zigbee-dual-switch.groovy

```diff
@@ -0,0 +1,140 @@
+/**
+*  Copyright 2015 SmartThings
+*
+*  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License. You may obtain a copy of the License at:
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+*  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+*  for the specific language governing permissions and limitations under the License.
+*
+*/
+
+metadata {
+    definition (name: "ZigBee Multi Switch", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.switch", runLocally: false, minHubCoreVersion: '000.019.00012', executeCommandsLocally: true, vid: "generic-switch" ) {
```

Comment:
The `minHubcoreVersion` and `executeCommandLocally` fields should be removed since it is not running locally.  When local support is added the `minHubcoreVersion` would be a different number as well.